components/git/epilogue.js

```diff
@@ -11,6 +11,7 @@ $ ncu-config set branch master   # Assuming you are landing commits on master
 $ git checkout master
 $ git node land --abort          # Abort a landing session, just in case
 $ git node land $PRID            # Start a new landing session
+$ git node land $URL             # Start a new landing session using the PR URL
```

Comment:
Just a suggestion, but it might make more sense to merge this into the line above, and just say that `$PRID` can be the PR number or the PR URL.

Reply:
@gibfahn hm, i don't think it makes big difference, i think it's fine to have them on separate line? i'm okay with changing it though. just let me know...

Reply:
I also don't think it makes a big difference, which is why it was just a suggestion. If you think it's fine as is, then that's fine by me.

Reply:
cool, then ready to land i guess?